src/neo-vm/ExceptionHandingContext.cs

```diff
@@ -0,0 +1,31 @@
+using System.Diagnostics;
+
+namespace Neo.VM
+{
+    [DebuggerDisplay("State={State}, TryPointer={TryPointer}, CatchPointer={CatchPointer}, FinallyPointer={FinallyPointer}, EndPointer={EndPointer}")]
+    public sealed class ExceptionHandingContext
+    {
+        public ExecutionContext ExecutionContext { get; private set; }
+        public int CatchPointer { get; private set; }
+        public int FinallyPointer { get; private set; }
+        public int EndPointer { get; private set; }
```

Comment:
Is it better to set this as an operand of `TRY`?

Reply:
No, `EndPointer` is dynamic, we may have multiple `ENDC`, `ENDT`.

```cs
try{
   if(xxx) { return a}  // ENDT#1
  if(yyy) { return b}  // ENDT#1
 ....
}catch{
}
```

Reply:
I don't think they are `ENDT`. They are `RET`.

Reply:
C# compiler will compile it to multiple `leave_s` -> `ENDT`. Need @lightszero to help explain it.

Reply:
EndT and Ret
leave_s ->EndT
after that，will got a ret

Reply:
After analyzing this pattern, I think `ENDT` and `ENDC` should be merged into one opcode. I have made corresponding changes.

Reply:
Should we add an offset operand to `ENDTRY`? Just like `leave.s`?

Reply:
> Should we add an offset operand to `ENDTRY`? Just like `leave.s`?

I think it's okay, `ENDTRY` like `leave`, and `ENDF` will jump to the offset of `ENDTRY` operand. If so, we'd better to add `ENDTRY_L` and rename it `ENDE/ENDE_L` (end exception)?

Reply:
How about emptying the eval stack when meet `ENDTRY`, just like `leave`.

>The leave instruction empties the evaluation stack and ensures that the appropriate surrounding finally blocks are executed. https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.opcodes.leave?view=netframework-4.8

Reply:
`ENDTRY` and `ENDTRY_L` are good names.

The vm doesn't need to empty the stack. The compiler can do it.

Reply:
@lightszero What do you think, could we add the operand? If possible, we'll add it.

Reply:
Yes I agree.I use EndT + JMP fornow.

Reply:
Done.